projects/admin/src/app/record/detail-view/acq-order-detail-view/order-lines/acq-order-lines.component.html

```diff
@@ -0,0 +1,74 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019 RERO
+  
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+  
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+  
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<h4 class="mt-4">{{ 'Order lines' | translate }}</h4>
+
+<div class="mb-2">
+  <a *ngIf="!order.permissions.cannot_update;"
+    class="btn btn-sm btn-primary"
+    [routerLink]="['/', 'records', 'acq_order_lines', 'new']"
+    [queryParams]="{ order: order.metadata.pid }"
+  >
+    <i class="fa fa-plus-square-o"></i> {{ 'Add' | translate }}
+  </a>
+</div>
+
+<ng-container *ngIf="orderLines">
+  <table class="table">
+    <thead>
+    <tr>
+      <th scope="col">#</th>
+      <th scope="col">{{ 'Description' |translate }}</th>
+      <th scope="col">{{ 'Amount' |translate }}</th>
+      <th scope="col">{{ 'Quantity' |translate }}</th>
+      <th scope="col">{{ 'Total amount' |translate }}</th>
+      <th scope="col">{{ 'Discount amount' |translate }}</th>
+      <th scope="col"></th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr *ngFor="let orderLine of orderLines; let i = index">
+      <th scope="row">{{ i + 1 }}</th>
+      <td>
+        <a [routerLink]="['/records', 'acq_order_lines', 'detail', orderLine.metadata.pid]">
+          {{ orderLine.metadata.description }}
+        </a>
+      </td>
+      <td>{{ orderLine.metadata.amount | currency:orderCurrency:'symbol':undefined:'fr-CH' }}</td>
+      <td>{{ orderLine.metadata.quantity }}</td>
+      <td>{{ orderLine.metadata.total_amount | currency:orderCurrency:'symbol':undefined:'fr-CH' }}</td>
+      <td>{{ orderLine.metadata.discount_amount | currency:orderCurrency:'symbol':undefined:'fr-CH' }}</td>
+      <td>
+        <button *ngIf="!orderLine.permissions.cannot_update" type="button" class="btn btn-link p-0"
+                [routerLink]="['/records', 'acq_order_lines', 'edit', orderLine.metadata.pid]">
+          <i class="fa fa-pencil"></i>
```

Comment:
`{{ 'edit' | translate }}` ?

Reply:
I would like to have just icon for the action